tools/cloud-build/hpc-toolkit-pr-validation.yaml

```diff
@@ -53,6 +53,20 @@ steps:
     time addlicense -check -ignore **/lysozyme-example/submit.sh . \
       || { echo "addlicense failed"; exit 1; }
     time make tests
+- id: ofe-virtual-env
+  waitFor:
+  - git-fetch-unshallow
```

Comment:
nit: To me it feels heavy handed to run this on every PR. Looking at recent builds it does not seem to have increased validation time, which is my reason to not block on this comment. I would propose creating a separate build trigger that is only triggered when this file is changed. You can use the `Included files` field in the trigger. https://cloud.google.com/build/docs/automating-builds/create-manage-triggers

Reply:
I agree with the claim and the recommendation. That said, the reason to put it in PR validation is that its standard output has been made visible to the public. The balance I chose to strike was to run it in parallel with the other (longer) tests.